lib/_http_server.js

```diff
@@ -200,8 +200,9 @@ ServerResponse.prototype._implicitHeader = function _implicitHeader() {
 };
 
 ServerResponse.prototype.writeHead = writeHead;
-function writeHead(statusCode, reason, obj) {
+function writeHead(statusCode, reason, headers) {
```

Comment:
Sorry, I was referring mainly to the parameter name change when asking 'why this change'?

Reply:
@mscdex so that "intellisense" matches the documentation and the compat layer.